controls/collectionview/drag-and-drop/reorder-grouped-items.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Reorder Grouped Items
+page_title: .NET MAUI CollectionView Documentation - Reorder Grouped Items
+description: Review the Telerik UI for .NET MAUI CollectionView Reorder Grouped Items.
+position: 1
+slug: collectionview-reorder-grouped-items
+tags: reorder, collectionview, drag-drop, grouping
+---
+
+# .NET MAUI CollectionView Reorder Grouped Items
+
+The .NET MAUI CollectionView provides the option to customize its drag and drop behavior and implement custom logic when items are dropped. This can be useful when you need to allow the users to move items between different groups in a grouped scenario. 
```

Comment:
```suggestion
The .NET MAUI CollectionView provides the option to customize its drag-and-drop behavior and implement custom logic when items are dropped. This can be useful when you need to allow the users to move items between different groups in a grouped scenario. 
```